### PR TITLE
41 implement oauth authentification

### LINKED ISSUE #41: Implement OAuth authentification
## Describe the feature
 Implement the authentification using differents services like facebook, google or apple

## TO DO
Add redirection to auth page front-side and communicate with back-end the user information

## Extra links (Optional)
_Add any other extra links_

## Screenshots (Optional)
_Add any screenshot_

### PR DESCRIPTION
**Pull request by CedricCORGE**
# New feature
Implement facebook login with redirection to home page if succeed

Closes #41

### CODE REVIEW

File: Web/package.json [hunk 1]
```diff
@@ -6,11 +6,14 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "boostrap": "^2.0.0",
```
Comment (31Nathan) [inline Web/package.json]: I don't think boostrap is necessary

**Review by 31Nathan — COMMENTED** [review]
Check the Web package

**Review by STom6 — APPROVED** [review]
ntr

**Review by osvegn — APPROVED** [review]
Look good

**Pull request merged**